products-2.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 	<head>
-		<title>All Products</title>
+		<title>Dry Skin Products</title>

```

Comment:
I decided that each page will be dedicated for each of the skin types. So page 2 will be dedicated to dry skin products etc.